assets/javascript/game.js

```diff
@@ -1,76 +1,159 @@
+
+  developerElement.childNodes[1].classList = 'developer__icon';
+  developerAttributes.childNodes[1].classList = 'developer__attribute';
+  developerAttributes.childNodes[3].innerHTML = 'Level: ' + developer.level;
+  developerAttributes.childNodes[3].classList = 'developer__attribute';
+  developerAttributes.childNodes[5].innerHTML = 'Lines/Click: ' + developer.performance();
+  developerAttributes.childNodes[5].classList = 'developer__attribute';
+  developerAttributes.childNodes[7].innerHTML = 'Price: ' + developer.price() + '$';
+  developerAttributes.childNodes[7].classList = 'developer__attribute';
+  developerAttributes.childNodes[9].innerHTML = 'Improve skills';
+
+  if (developersElement.lastElementChild === developerElement && developerPosition < developers.length-1) {
+    addNewMenuDeveloper(developers[developerPosition+1]);
   }
 }
 
-function click() {
-  writtenLines = writtenLines + click_value;
+function officeClick() {
+  writtenLines = writtenLines + mainDeveloper.performance();
+
   updatePerformance();
 }
 
-function addNewDeveloper(developer) {
-  developer.level = 1;
-  employers.innerHTML += '<img alt="' + developer.name + '" class="character" src="assets/images/employers/' + developer.imageName + '.png">'
+function addNewMenuDeveloper(developer) {
+  developersElement.insertAdjacentHTML("beforeend",`<div id=${developer.id} class="developer__menu">
+        <div class="developer__icon--inactive">
+          <img alt=${developer.name} class="developer" src="assets/images/employees/${developer.id}.png">
+        </div>
+        <div class="developer__attributes">
+          <p class="developer__attribute--inactive">Name: ${developer.name}</p>
+          <p class="developer__attribute--inactive">Level: ${developer.level}</p>
+          <p class="developer__attribute--inactive">Lines/Click: 0</p>
+          <p class="developer__attribute--inactive">Price: ${developer.price()}$</p>
+          <button class="developer__button">Hire this developer</button>
+        </div>
+      </div>`);
+
+  developersElement
+    .lastElementChild
+    .lastElementChild
+    .lastElementChild
+    .addEventListener("click", improveDeveloper);
+  developersElement
+    .lastElementChild
+    .firstElementChild
+    .lastElementChild
+    .addEventListener("click", showDeveloperInfo);
 }
+
+function showDeveloperInfo(event) {
+  let developerId = event.target.parentNode.parentNode.id;
+  let developer = developers.find(developer => developer.id === developerId);
+  customizeModal(developer);
+  openModal();
+}
+
+function preventZoom(event) {
+  let secondTouch = event.timeStamp;
+  let firstTouch = event.currentTarget.dataset.lastTouch || secondTouch;
+  let timeDifference = secondTouch - firstTouch;
+  let fingers = event.touches.length;
+  event.currentTarget.dataset.lastTouch = secondTouch;
+
+  if (!timeDifference || timeDifference > 500 || fingers > 1) return; // not double-tap
+
+  event.preventDefault();
```

Comment:
This shouldn't be at top level of this function? I mean, in some specific cases if the browser will start execute the default behavior until you call `preventDefault` function

Reply:
`Calling preventDefault during any stage of event flow cancels the event, meaning that any default action normally taken by the implementation as a result of the event will not occur.`

What browser do you mean? It should behave the same no matter where you put it according to some research, I just put it on the end to follow convention, since it replaces `return false`

Reply:
I'm not sure about this event, but `preventDefault` in specific events should to be done at top level of the function, eg: submit event, if you decide to prevent after of doing some stuff could be that browser already processed the default event

Reply:
The idea is just move to top 
```js
event.preventDefault();
..
```